### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by taty2010**
Adding that this repo is now archived

### CODE REVIEW

**Comment by taty2010** [comment]
@MarcL and @stephan1echung I ended up only needing to make a small change to the Readme since the Redirects notice was already added. If you don't think this is enough let me know.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ boilerplates.
 
 ## Important Notice
 
-:warning: [templates.netlify.com](https://templates.netlify.com/) is now being redirected to relevant GitHub repos, [jamstack.org/](https://jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
+:warning: the [templates.netlify.com](https://templates.netlify.com/) repo is now archived and currently being redirected to relevant GitHub repos, [jamstack.org/](https://jamstack.org/), or [www.netlify.com](https://www.netlify.com/) pages. If you are interested in adding a template to [netlify.com/integrations/templates/](https://www.netlify.com/integrations/templates/), please reach out to templates@netlify.com
```
Comment (stephan1echung) [inline README.md:8]: I'd change this ever so slightly to `The [templates.netlify.com](https://templates.netlify.com/) repo is now archived, and the site is currently being redirected to relevant...`

**Review by stephan1echung — CHANGES_REQUESTED** [review]
Left a super smol comment to update copy ever so slightly!

**Review by stephan1echung — APPROVED** [review]
LGTM :shipit:

**Pull request merged**